backend-ci/tests/Services/ProductVariantServiceTest.php

```
@@ -0,0 +1,368 @@
+        $removeResult = $this->service->removeAttributeFromVariant($vid, $attrId);
+        $this->assertTrue($removeResult['success']);
+
+        $getResultEmpty = $this->service->attributeValues($vid);
+        $this->assertCount(0, $getResultEmpty['data']);
+    }
+
+    public function test_deletedList(): void
+    {
+        $pid = $this->seedProduct(['code' => 'P1']);
+        $vid1 = $this->seedVariant($pid, 'SKU1');
+        $vid2 = $this->seedVariant($pid, 'SKU2');
```

PHPMD (2.15.0)

Avoid unused local variables such as '$vid2'. (undefined)

(UnusedLocalVariable)